in-work/lln_clt.md

```diff
@@ -252,6 +411,7 @@ $\mu$ and common variance $\sigma^2 \in (0, \infty)$, then
 \quad \text{as} \quad
 n \to \infty
 ```
+````
 
 Here $\stackrel { d } {\to} N(0, \sigma^2)$ indicates [convergence in distribution](https://en.wikipedia.org/wiki/Convergence_of_random_variables#Convergence_in_distribution) to a centered (i.e, zero mean) normal with standard deviation $\sigma$.
```

Comment:
``i.e,`` should be ``i.e.,``.

Please refer to: https://en.wikipedia.org/wiki/The_Elements_of_Style